src/content/en/2020/resource-hints.md

```diff
@@ -110,31 +110,38 @@ More and more web pages are using the main resource hints, and in 2020 we are se
 
 {{ figure_markup(
   image="adoption-of-resource-hints.png",
-  caption="Adoption of resource hints",
-  description="TODO",
+  caption="Adoption of resource hints.",
+  description="A bar chart of the rate of resource hint adoption broken down by hint type and form factor. For desktop, 33% of pages use the `dns-prefetch` resource hint, 18% use `preload`, 8% use `preconnect`, 3% use `prefetch`, and less than 1% use `prerender`. Mobile is similar, except that `dns-prefetch` has 34% usage (1% higher), and `preconnect` has 9% (1% higher).",
```

Comment:
Sorry, @Zizzamia I should have linked you to the "description" [guidance](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide#writing-good-figure-descriptions). I reworded them, make sure you think they are appropriate.

Reply:
Oh I seeeee! Thank you for taking care of it.